### ISSUE TITLE
Documentation - PDF Error - Missing Text Equivalent for Images

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12708]
This will be the spike ticket for writing documentation for fixing the missing text equivalent for images issue
